doc/source/how_to/config/customization.ipynb

```diff
@@ -54,56 +102,112 @@
    "id": "eaaaa227-e941-4516-b451-69bbbb408b06",
    "metadata": {},
    "source": [
-    "The configuration may also be accessed as a Python `dict`:"
+    "The client/server configuration can also be accessed individually as a Python `dict` as follows"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "id": "55bc1913-5507-417d-a6af-7bb989256f4c",
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "{'sdk': {'log_dir': '/var/home/user/.cache/covalent',\n",
+       "{'sdk': {'config_file': '/home/venkat/.config/covalent/covalent.conf',\n",
+       "  'log_dir': '/home/venkat/.cache/covalent',\n",
        "  'log_level': 'warning',\n",
        "  'enable_logging': 'false',\n",
-       "  'executor_dir': '/var/home/user/.config/covalent/executor_plugins'},\n",
+       "  'executor_dir': '/home/venkat/.config/covalent/executor_plugins',\n",
+       "  'no_cluster': 'false'},\n",
+       " 'server': {'address': 'localhost', 'port': 48008},\n",
        " 'dispatcher': {'address': 'localhost',\n",
-       "  'port': 45232,\n",
-       "  'cache_dir': '/var/home/user/.cache/covalent',\n",
+       "  'port': 48008,\n",
+       "  'cache_dir': '/home/venkat/.cache/covalent',\n",
        "  'results_dir': 'results',\n",
-       "  'log_dir': '/var/home/user/.cache/covalent'},\n",
+       "  'log_dir': '/home/venkat/.cache/covalent',\n",
+       "  'db_path': '/home/venkat/.local/share/covalent/dispatcher_db.sqlite'},\n",
+       " 'dask': {'cache_dir': '/home/venkat/.cache/covalent',\n",
+       "  'log_dir': '/home/venkat/.cache/covalent',\n",
+       "  'mem_per_worker': 'auto',\n",
+       "  'threads_per_worker': 1,\n",
+       "  'num_workers': 16},\n",
+       " 'workflow_data': {'storage_type': 'local',\n",
+       "  'base_dir': '/home/venkat/.local/share/covalent/workflow_data'},\n",
        " 'user_interface': {'address': 'localhost',\n",
-       "  'port': 45232,\n",
-       "  'log_dir': '/var/home/user/.cache/covalent'},\n",
+       "  'port': 48008,\n",
+       "  'dev_port': 49009,\n",
+       "  'log_dir': '/home/venkat/.cache/covalent'},\n",
        " 'executors': {'local': {'log_stdout': 'stdout.log',\n",
        "   'log_stderr': 'stderr.log',\n",
-       "   'cache_dir': '/tmp/covalent'}}}"
+       "   'cache_dir': '/home/venkat/.cache/covalent'},\n",
+       "  'dask': {'log_stdout': 'stdout.log',\n",
+       "   'log_stderr': 'stderr.log',\n",
+       "   'cache_dir': '/home/venkat/.cache/covalent'},\n",
+       "  'remote_executor': {'poll_freq': 15,\n",
+       "   'remote_cache': '.cache/covalent',\n",
+       "   'credentials_file': ''}}}"
       ]
      },
-     "execution_count": 1,
+     "execution_count": 5,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "import covalent as ct\n",
-    "ct.get_config()"
+    "from covalent._shared_files.config import CMType\n",
```

Comment:
Shouldn't we expose `CMType` to the main `covalent` namespace instead of asking users to dig into internal files?

Reply:
@venkatBala can you split the customization.ipynb fix into a separate PR? That could be merged more quickly.

Reply:
@scottwn based on what we discussed last week... should I isolate the changes to `customization.ipynb` as part of another PR... I recall we discussed to have it part of the same one... as the changes are quite minor... thoughts?